### PR TITLE
[cli] support package.json based turbo config in vc build automatic build/install command feature

### PR DESCRIPTION
**Pull request by Ethan-Arrowood**
### Related Issues

Adds support for package.json based turbo configuration. 

Includes test.

Confirmed with Turbo that `turbo.json` takes precedence over `package.json` based config so that is how it is processed here.

#### Tests

- [ ] The code changed/added as part of this PR has been covered with tests
- [ ] All tests pass locally with `yarn test-unit`

#### Code Review

- [ ] This PR has a concise title and thorough description useful to a reviewer
- [ ] Issue from task tracker has a link to this PR

### CODE REVIEW

**Review by EndangeredMassa — APPROVED** [review]

**Review by TooTallNate — APPROVED** [review]

**Review by cb1kenobi — APPROVED** [review]

**Comment by Ethan-Arrowood** [comment]
Removing auto merge and adding another fix to this PR :)

File: packages/cli/src/util/build/monorepo.ts [hunk 1]
```diff
@@ -55,21 +55,40 @@ export async function setMonorepoDefaultSettings(
...
-        'Missing required `build` pipeline in turbo.json. Skipping automatic setting assignment.'
+        'Missing required `build` pipeline in turbo.json or package.json Turbo configuration. Skipping automatic setting assignment.'
       );
       return;
     }
 
     setCommand(
       'buildCommand',
-      `cd ${relativeToRoot} && npx turbo run build --filter=${projectName}...`
+      `cd ${relativeToRoot} && npx turbo run build --filter={./${relative(
```
Comment (TooTallNate) [inline packages/cli/src/util/build/monorepo.ts:88]: Do we need to worry about Windows here and have `.\\`?

Comment (Ethan-Arrowood) [reply packages/cli/src/util/build/monorepo.ts:88]: Great question - i assume CI will fail if thats the case. But let me ask Turbo right now too

Comment (Ethan-Arrowood) [reply packages/cli/src/util/build/monorepo.ts:88]: https://github.com/vercel/vercel/actions/runs/3483582850/jobs/5827228329#step:8:2934 ironically CI timeout on an Nx test, but all the Turbo ones seemed to pass on Windows

Comment (EndangeredMassa) [reply packages/cli/src/util/build/monorepo.ts:88]: I think we should always `path.join` unless we specifically have a good reason not to. It's more likely to save us the trouble, right?

Comment (TooTallNate) [reply packages/cli/src/util/build/monorepo.ts:88]: It's kinda annoying, but `path.join()` will strip the `./`:

```
> path.join('.', 'foo')
'foo'
```

**Pull request merged**